src/header.rs

```diff
@@ -95,16 +92,15 @@ mod tests {
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
     #[cfg_attr(not(target_arch = "wasm32"), test)]
     fn symmetric_header_v1() {
-        let digest = multihash::Code::Blake3_256.digest(b"test");
-        let cid = Cid::new_v1(DagCborCodec.into(), digest);
+        let digest = multihash_codetable::Code::Blake3_256.digest(b"test");
+        let cid = Cid::new_v1(0x71, digest);
```

Comment:
couldn't get the code from `multihash_codetable` so I just copied it..

Reply:
As you use `ipld-core` already, you should be able to import the `Codec` trait from it and then be able to use `DagCborCodec::CODE` (I haven't tested it but this how it should work).

Though I think it would make sense to make that constant maybe public directly in `serde_ipld_dagcbor`, withou the need to import the `Codec` trait.

Reply:
that doesn't work

> As you use `ipld-core` already, you should be able to import the `Codec` trait from it and then be able to use `DagCborCodec::CODE`

I tried that, it throws an error of not knowing `T`